.github/workflows/pull-request.yml

```
@@ -0,0 +1,265 @@
+        with:
+          node-version: '22.x'
+
+      - name: resolve dependency
+        run: npm install -g @graphql-inspector/cli
+      
+      - name: Clone API Repository
+        run: |
+          # Retrieve the complete branch name directly from the GitHub context
+          FULL_BRANCH_NAME=${{ github.base_ref }}
+          echo "FULL_Branch_NAME: $FULL_BRANCH_NAME"
+          
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)